etc/encoding/opencast.properties

```
@@ -255,3 +255,11 @@ profile.text-analysis.http.suffix = .#{time}.png
 profile.text-analysis.http.ffmpeg.command = -ss #{time} -i #{in.video.path} \
   -vf "fps=1,scale=1280:720:force_original_aspect_ratio=decrease,pad=1280:720:-1:-1 [in]; color=black:s=1280x720:r=1:d=1 [bg]; [bg][in] overlay=0:0 [mux]; [mux] curves=preset=increase_contrast,format=pix_fmts=gray [out]" \
   -frames:v 1 #{out.dir}/#{out.name}#{out.suffix}
+
+# Encode audio-only to mp3
+profile.audio-to-mp3.http.name = audio-to-mp3
+profile.audio-to-mp3.http.input = audio
+profile.audio-to-mp3.http.output = audio
+profile.audio-to-mp3.http.suffix = -audio.mp3
+profile.audio-to-mp3.http.mimetype = audio/mpeg
+profile.audio-to-mp3.http.ffmpeg.command = -i #{in.video.path} -vn -map_metadata -1 -sn -acodec libmp3lame -ab 128k -ar 44100 -ac 2  #{out.dir}/#{out.name}#{out.suffix}
```

Instead of still using mp3, why not use AAC in an `.m4a` instead? That should work everywhere today and is more efficient.

Also, for lecture capture I would usually suggest just going with mono (`-ac 1`) or not modify the channel count if you want to keep it.

Something like:
```suggestion
profile.audio-to-mp3.http.suffix = -audio.m4a
profile.audio-to-mp3.http.mimetype = audio/mp4
profile.audio-to-mp3.http.ffmpeg.command = -i #{in.video.path} -vn -map_metadata -1 -sn -c:a aac -b:a 64k -ar 44100  #{out.dir}/#{out.name}#{out.suffix}
```

---

@lkiesow limiting bitrate of an audio encoder without setting the channels is a bad idea because the bitrate per channel will be `<bitrate>/<#channels>`. Example for an stereo stream: `64k/2=32k per channel`. This might be ok but unexpected. I would suggest `-b:a 64k -ac1` or `-b:a 128k -ac 2` for high quality stream.